design/definitions.go

```diff
@@ -1054,6 +1054,20 @@ func (a *AttributeDefinition) IsInterface(attName string) bool {
 	return att.Type.Kind() == AnyKind
 }
 
+// IsFile returns true if the field generated for the given attribute has
```

Comment:
Ideally the DSL and design functions should not be defined in terms of generated Go code since the design describes the API "in the absolute". So maybe something like:
```
IsFile returns true if the attribute is of type File or if any its children attributes (if any) is.
```

Reply:
I fixed it. https://github.com/goadesign/goa/pull/1612/commits/9cf3d77c29560a7519072228ead52a91b1655960